cli/cmd/onboard_service.go

```diff
@@ -125,6 +98,36 @@ keptn onboard service SERVICENAME --project=PROJECTNAME --chart=HELM_CHART.tgz
 	},
 }
 
+func doOnboardServicePreRunChecks(args []string) error {
+	_, _, err := credentialmanager.NewCredentialManager(false).GetCreds(namespace)
+	if err != nil {
+		return errors.New(authErrorMsg)
+	}
```

Comment:
To be honest, I wouldn't have expected the credential manager stuff in the `doOnboardServicePreRunChecks` function, but in `RunE` (as it is in `create_project`). But I'm fine if it stays in here too. What's your thought on this @bacherfl ?

Reply:
True, that credential manager check didn't need to be there since the credential manager is checked again in the `RunE` method.

Reply:
Oh yeah, I see it :) Thanks!